src/wallet/init.cpp

```diff
@@ -15,6 +15,8 @@
 std::string GetWalletHelpString(bool showDebug)
 {
     std::string strUsage = HelpMessageGroup(_("Wallet options:"));
+    strUsage += HelpMessageOpt("-addressstyle", _("What style addresses to use (\"legacy\", \"p2sh\", \"bech32\", default: \"p2sh\")"));
+    strUsage += HelpMessageOpt("-changestyle", _("What style change to use (\"legacy\", \"p2sh\", \"bech32\"), default: same as -addressstyle"));
```

Comment:
I'm not  native speaker, but "What style change to use" seems like broken English to me. Shouldn't it be something like "Which style of change addresses to use"?

Reply:
Native English speaker here.  Here's how I'd write these two:

"Receiving address type (...)"
"Change address type (...)"

Legacy/p2sh/bech32 feel like types to me, not styles.  Though at that point you'd have to change the command line parameters as well.

Reply:
Changed to address_type and change_type, and edited the comment language.